tests/unit/test_pipeline_graph.py

```diff
@@ -6,11 +6,32 @@
 from easylink.graph_components import InputSlot
 from easylink.pipeline_graph import PipelineGraph
 from easylink.utilities.validation_utils import validate_input_file_dummy
+from tests.unit.conftest import COMBINED_IMPLEMENTATION_CONFIGS
+
+
+def check_nodes_and_edges(pipeline_graph, expected_nodes, expected_edges):
```

Comment:
Nit: I'd make this a private function to clarify it is a helper function or put it in a helper function section at the bootom.

Reply:
added a helper section